docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       - "5431:5432"
     volumes:
       - hatchet_postgres_data:/var/lib/postgresql/data
+    shm_size: 4g
```

Comment:
i'm pretty sure this is not needed, but was hitting a memory limit when reindexing the old step run compound index.